internal/core/items.rs

```diff
@@ -437,32 +437,18 @@ pub struct TouchArea {
 }
 
 impl TouchArea {
-    fn start_clicked_timer(self: Pin<&Self>, item_rc: &ItemRc) -> bool {
+    fn start_clicked_timer(self: Pin<&Self>) {
```

Comment:
Why do we still need this timer, can it not be removed? We already have the click count from the event (`click_count` in `MouseEvent:: Released `  )